x-pack/plugins/security_solution/server/lib/security_integrations/cribl/routes/index.ts

```diff
@@ -17,6 +17,12 @@ export const getFleetManagedIndexTemplatesRoute = (router: IRouter) => {
     .addVersion(
       {
         version: '1',
+        security: {
+          authz: {
+            enabled: false,
+            reason: 'Route delegates authorization to the scoped ES Client',
```

Comment:
```suggestion
            reason: 'This route delegates authorization of the current user to the Elasticsearch index template API.',
```

This might be a bit pedantic, but IIUC the authorization is delegated to Elasticsearch, rather than to the API client.

Reply:
Fine by me. I grabbed that text from a slack discussion about a similar situation, but this works as well